packer/scripts/ami-cleanup.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+import boto3
+from botocore.exceptions import ClientError
+
+REGIONS = [
+    "us-west-2",
+    "eu-central-1",
+    "ap-southeast-1",
+    "eu-west-2"
+]
+
+EPOCH_IMAGE_NAME = "epoch-ubuntu-16.04"
```

Comment:
I see hardcoded prefix in other repo too - i.e. [here](https://github.com/aeternity/infrastructure/blob/e6bd0ba7063d4f3d085a1af7ac90ee96f3c66ba0/terraform/modules/cloud/aws/deploy/fleet/main.tf#L9-L23) that is not removed by https://github.com/aeternity/infrastructure/pull/115/files

Not sure how you will prevent this hardcoded string to get out of sync across the two repos (grep not possible anymore). I do not have solution though so ok.